src/app/beta/orgs/[orgId]/canvassassignments/[canvassAssId]/sessions/route.ts

```diff
@@ -0,0 +1,131 @@
+import mongoose from 'mongoose';
+import { NextRequest, NextResponse } from 'next/server';
+
+import { AreaModel, CanvassAssignmentModel } from 'features/areas/models';
+import { ZetkinCanvassSession } from 'features/areas/types';
+import asOrgAuthorized from 'utils/api/asOrgAuthorized';
+import { ZetkinPerson } from 'utils/types/zetkin';
+
+type RouteMeta = {
+  params: {
+    canvassAssId: string;
+    orgId: string;
+  };
+};
+
+export async function GET(request: NextRequest, { params }: RouteMeta) {
+  return asOrgAuthorized(
+    {
+      orgId: params.orgId,
+      request: request,
+      roles: ['admin'],
+    },
+    async ({ apiClient, orgId }) => {
+      await mongoose.connect(process.env.MONGODB_URL || '');
+
+      const model = await CanvassAssignmentModel.findOne({
+        _id: params.canvassAssId,
```

Comment:
Maybe `orgId` here too?

Reply:
Yeah, there should be, for sure! Lets try to remember to fix it in the beta API, and otherwise it will definitely be solved in the real API.